dev-db/percona-xtrabackup/metadata.xml

```diff
@@ -6,6 +6,11 @@
     <name>Tomáš Mózes</name>
     <description>Proxy maintainer</description>
   </maintainer>
+  <maintainer type="person">
+    <email>manwe@manwe.pl</email>
+    <name>Michał Zając</name>
+    <description>Proxy maintainer</description>
```

Comment:
We no longer use those descriptions, so if you could drop all of them, would be appreciated. Basically, descriptions are to be used to say how to assign bugs.